autobuild/app.py

```diff
@@ -71,12 +71,17 @@ def processcustom(customlist):
     used_ip = {}
     to_del_index = []
     daemons_list = []
-    daemonFiles = {}
+    configFiles = {}
+    binFiles = {}
     rpc_threads = 0
     manifest_config = autoconfig.load_template(autoconfig.manifest_content(BRANCHPATH))
     manifest = json.loads(Template(manifest_config).render())
     for blockchain in manifest:
-        daemonFiles[blockchain['ticker']] = blockchain['conf_name']
+        configFiles[blockchain['ticker']] = blockchain['conf_name']
+        if 'daemon_stem' in blockchain:
+            binFiles[blockchain['ticker']] = blockchain['daemon_stem'] + 'd'
+        else:
+            binFiles[blockchain['ticker']] = blockchain['conf_name'].split('.conf')[0] + 'd'
```

Comment:
I know there aren't many comments in this code, but it might be nice to add a comment here (lines 79-84), giving an example of a non-standard blockchain name which this enhancement will allow us to accommodate.
I guess the new `daemon_stem` field of `manifest-latest.json` should also be documented in the README of the `blockchain-configuration-files` branch, no?

Reply:
Added minimal doc.

Yeah, there'll be a couple more PRs forthcoming for the related `blockchain-configuration-files` and `dockerimages` changes